static/docs/commands-reference/get-url.md

```diff
@@ -0,0 +1,176 @@
+# get-url
+
+Download or copy file or directory from any supported URL (for example
+`http://`, `s3://`, `ssh://`, and other protocols) or local directory to the
+<abbr>workspace</abbr>.
```

Comment:
it's actually not necessarily the workspace. It's more like a wget/rclone/curl. You don't even need a repo to get the data. The same for regular get. We might even even make a direct reference to import-url - that it's the same but does not require a repo, does not create DVC-files. It's single purpose is just download a data artifact.

Reply:
Got it! Good to know, will update.

Reply:
Updated in 54ca3ca. BTW re

> We might even even make a direct reference to import-url - that it's the same but does not require a repo, does not create DVC-files. Its single purpose is just download a data artifact.

And also `get` commands don't support `remote://` URLs right? See https://github.com/iterative/dvc/pull/2160#pullrequestreview-252712391

The import* vs get* refs are not that similar though, every section is different. I don't know if just making a ref from get to import will suffice as documentation.

Reply:
Definitely not enough. Just as an additional reference if you see it makes sense. Since they are similar after all.

re remote:// - yes, like @efiop mentioned in that comment.

Reply:
Sure, I did add the links between import<->import-get as well as import<->get, and all other related commands (: